### DISCUSSION TITLE
Upgrade path from 3.3 to 4.1

### DISCUSSION THREAD

**Discussion by jamiefuller96** [fossology/fossology#2254] [Q&A] [answered]
Hello,

Can someone provide information on the best upgrade path from version 3.3.0 to 4.1.0.16? Does it need to be incremental, can I just upgrade straight to 4.1.0?

Thanks

**Comment by shaheemazmalmmd** [comment, marked as answer]
Hello @jamiefuller96 

If you are doing a upgrade through source, Would recommend you to do "straight upgrade". In case of Docker you can choose "incremental upgrade".
